### ISSUE TITLE
10-10CG - Secondary Caregiver Multiple Response MVP

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#92337]
## Benefit to the Veteran
As a Veteran and/or Caregiver, I want to see the Secondary Caregivers listed in one place, so that I can remember who I have entered already and edit any details easily.

## Hypothesis or Bet
**If** _we implement the Multi-Response pattern to the Caregivers_ **then** _applicants will be able to see the list of Secondary Caregivers they have entered in that session and edit any details as needed_.

## OKR
_Which Objective / Key Result does this epic push forward?_

## MVP Scope
- Since an applicant is able to enter up to two Secondary Caregivers, we would like to implement the multi-response pattern to handle the entries.  This will display a list of Secondary Caregivers to the applicant, allowing them to see who they have already listed, and edit/remove any details as needed.
- In Scope:
     - MVP will consist of adding the multi-response pattern to the Secondary Caregivers section.  
- Out of Scope: 
     - Including Primary CG in the multi-response list
     - Changes to how data is formatted for the generated PDF
     - Changes to how the data is formatted to send to Mulesoft/CARMA
- Design Explorations are needed before we can continue work on this.
- We may take this through Collab Cycle
- There are no external dependencies to identify

## Resources
- [Product brief - 10-10CG](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/caregivers/10-10CG%20Caregiver%20application%20product-outline.md)
- [Initiative brief - Secondary Caregiver Multiple Response](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/caregivers/Multi-Response/Secondary%20Caregiver%20Multiple%20Response%20MVP/Secondary%20Caregiver%20Multiple%20Response%20MVP%20-%20Initiative%20Brief.md)  
- [Design Exploration](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=1012-27459&node-type=SECTION&t=VVrpImQGrq4BuYrZ-0)
- Screenshots
- Reference docs/tickets where we have this kind of work documented
- Etc.
---
## Acceptance Criteria/Definition of Done
-  Applicants will be able to enter up to two Secondary Caregivers, see them listed in cards with the ability to edit their details or remove them completely.  

---

```[tasklist]
### Tasks
- [ ] Research
- [ ] Design
- [ ] Team review & signoff
- [ ] UI update
- [ ] Team review & signoff
- [ ] BE update
- [ ] QA
- [ ] Team review & signoff
- [ ] End to end testing
- [ ] Release Plan
- [ ] Cypress test suite
- [ ] Team review & signoff
- [ ] Launch to Prod
- [ ] Update Source of Truth
```

---
## How to configure this issue
- [x] **Labeled with Team** (`1010-team`)
- [x] **Labeled with Form** (`1010-ez`, `1010-ezr`, `1010-cg`)
- [x] **Labeled with Project** (`facilities-cg`, `registration`,  `toxic-exposure`, etc.)

**Comment by hdjustice** [comment]
PO Patrick Bateman is not interested in launching this feature.  There is no need to add the multi-response/List&Loop to the CG form
